contrib/spice-gtk/template.py

```diff
@@ -0,0 +1,67 @@
+pkgname = "spice-gtk"
+pkgver = "0.42"
+pkgrel = 0
+build_style = "meson"
+configure_args = [
+    "-Dcoroutine=libucontext",
```

Comment:
gthread

Reply:
Done.

Curious though, why'd you pick gthread over libucontext?  
performance? stability? simplicity? or just to reduce dependencies? or something else?